### PR TITLE
+ support for nuxt3

### PR DESCRIPTION
**Pull request by mikespinks0401**
Added support for Nuxt3.

After modifying the config file to upload my static Nuxt 3 apps, I realized it would be beneficial to fork this action and add Nuxt 3 support for a smoother experience.

This is my first pull request, and I would appreciate any feedback on how I can improve it.

### CODE REVIEW

File: src/set-pages-config.js [hunk 1]
```diff
@@ -85,7 +101,7 @@ function setPagesConfig({ staticSiteGenerator, generatorConfigFile, siteUrl }) {
       core.warning(
         `Unsupported configuration file extension. Currently supported extensions: ${SUPPORTED_FILE_EXTENSIONS.map(
           ext => JSON.stringify(ext)
-        ).join(', ')}`,
+        ).join(', ')} recieved: something`,
```
Comment (yoannchaudet) [inline src/set-pages-config.js:104]: Looks like a copy/paste misap? 

```suggestion
        ).join(', ')}`,
```

File: src/index.js [hunk 2]
```diff
@@ -10,10 +10,9 @@ const outputPagesBaseUrl = require('./output-pages-base-url')
 async function main() {
   try {
     const { githubToken, enablement, staticSiteGenerator, generatorConfigFile } = getContext()
-
     const pageObject = await findOrCreatePagesSite({ githubToken, enablement })
     const siteUrl = new URL(pageObject.html_url)
-
+    core.debug('Site url:' + siteUrl + "or " + pageObject.html_url + " : " + pageObject.url )
```
Comment (yoannchaudet) [inline src/index.js:15]: We probably want to remove that.

**Comment by yoannchaudet** [comment]
What did not work with our `nuxt` option to justify having a dedicated `nuxt3` option? Also I see some `ts` extension here and there but we don't have a TypeScript parser in this repo as far as I am aware of 🤔

**Comment by mikespinks0401** [comment]
I believe I jumped the gun late at night. I will further test and implement. Thank you for your response.

**Pull request closed without merging**